package.json

```diff
@@ -23,16 +23,17 @@
   ],
   "main": "lib/canvassmith",
   "engines": {
-    "node": ">= 0.8.0"
+    "node": ">= 0.10.0"
   },
   "scripts": {
     "test": "mocha --timeout 60000"
   },
   "dependencies": {
-    "obj-extend": "~0.1.0",
-    "temporary": "0.0.5",
+    "canvas": "^1.2.2",
     "async": "~0.2.7",
-    "canvas": "~1.1.1"
+    "canvas": "^1.2.3",
```

Comment:
Please use `~` as the semver range, I prefer more stability with more releases than things breaking unpredictably.